app/src/main/java/org/gdg/frisbee/android/activity/MainActivity.java

```diff
@@ -175,6 +176,29 @@ protected void onResume() {
             GooglePlayServicesUtil.getErrorDialog(playServiceStatus, this, PLAY_SERVICE_DIALOG_REQUEST_CODE).show();
         }
         checkHomeChapterValid();
+        checkOrganizer();
+    }
+
+    @Override
+    public void onConnected(Bundle bundle) {
+        super.onConnected(bundle);
+        checkOrganizer();
+    }
+
+    private void checkOrganizer() {

```

Comment:
just the naming, seems too much organizers and checks.

the method name could be "updatePages"
OrganizerChecker.OrganizerResponseHandler should be just OrganizerChecker.ResponseHandler